Source/Core/Core/Slippi/SlippiPlayback.cpp

```diff
@@ -283,6 +281,21 @@ void SlippiPlaybackStatus::SeekThread()
 	INFO_LOG(SLIPPI, "Exit seek thread");
 }
 
+// Set isHardFFW and update OC settings to speed up the FFW
+void SlippiPlaybackStatus::setHardFFW(bool enable) {
+	isHardFFW = enable;
+	if (isHardFFW)
+	{
```

Comment:
Might be nice to set the prev variables here too in the case where someone changes them while the game is running? In order to do that though you might need to make sure you only trigger the if block when first transitioning to hardFFW so that the prev variables don't get overwritten with the FFW values.